src/components/PeopleTableHeaders.tsx

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import { Link, useSearchParams } from 'react-router-dom';
+import { getSearchWith } from '../utils/searchHelper';
+
+export enum Headers {
+  name = 'Name',
+  sex = 'Sex',
+  born = 'Born',
+  died = 'Died',
+  mother = 'Mother',
+  father = 'Father',
+}
+
+export enum Order {
+  asc = 'asc',
+  desc = 'desc',
+}
+
+export interface PeopleTableHeadersProps {
+  label: Headers;
+  hasLink?: boolean;
+  order?: Order;
+}
+
+export function PeopleTableHeaders({
+  label,
+  hasLink,
+}: PeopleTableHeadersProps) {
+  const [searchParams] = useSearchParams();
+
+  const sort = searchParams.get('sort') || '';
+  const order = searchParams.get('order') || '';
+  const orderValue = order as Order;
+
+  function handleSortOrder() {
+    switch (true) {
+      case sort && sort === label.toLowerCase() && !order:
+        return getSearchWith(searchParams, {
+          sort: label.toLowerCase(),
+          order: Order.desc,
+        });
```

Comment:
In the `handleSortOrder` function, the first case checks if `sort` is equal to `label.toLowerCase()` and `order` is falsy. This will always set the order to `desc` when the sort is initially applied. Ensure this logic aligns with the desired behavior for sorting when no order is specified.